stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -4,39 +4,109 @@ import com.google.gson.annotations.SerializedName
 import java.util.Date
 
 public data class Config(
+
+    /**
+     * Date of channel creation.
+     */
     @SerializedName("created_at")
-    var created_at: Date? = null,
+    val created_at: Date? = null,
 
+    /**
+     * Date of last channel update.
+     */
     @SerializedName("updated_at")
-    var updated_at: Date? = null,
+    val updated_at: Date? = null,
+
+    /**
+     * The name of the channel type must be unique per application
+     */
+    val name: String = "",
 
+    /**
+     * Controls if typing indicators are shown. Enabled by default.
+     */
     @SerializedName("typing_events")
     var isTypingEvents: Boolean = false,
 
+    /**
+     * Controls whether the chat shows how far you’ve read. Enabled by default.
+     */
     @SerializedName("read_events")
     var isReadEvents: Boolean = false,
 
+    /**
+     * Determines if events are fired for connecting and disconnecting to a chat. Enabled by default.
+     */
     @SerializedName("connect_events")
-    var isConnectEvents: Boolean = false,
+    val isConnectEvents: Boolean = false,
 
+    /**
+     * Controls if messages should be searchable (this is a premium feature). Disabled by default.
+     */
     @SerializedName("search")
-    var isSearch: Boolean = false,
+    val isSearch: Boolean = false,
 
+    /**
+     * If users are allowed to add reactions to messages. Enabled by default.
+     */
     @SerializedName("reactions")
-    var isReactionsEnabled: Boolean = false,
+    val isReactionsEnabled: Boolean = false,
```

Comment:
> Enabled by default.
> `= false`

This looks really confusing. I know these default values will probably never be used, but should we attempt to align them with what the default behaviour is?

Reply:
Need to check that our tests will make sense after that. I guess everything is `false` by default here and we are testing that updated config has all fields set to `true`